ziti/constants/constants.go

```diff
@@ -58,62 +60,64 @@ const (
 // Env Var Constants
 const (
 	ZitiHomeVarName        = "ZITI_HOME"
-	ZitiHomeVarDescription = "Root home directory for Ziti related files"
+	ZitiHomeVarDescription = "Root home directory for Ziti-related files"
 
 	PkiCtrlCertVarName                               = "ZITI_PKI_CTRL_CERT"
-	PkiCtrlCertVarDescription                        = "Path to Identity Cert for Ziti Controller"
+	PkiCtrlCertVarDescription                        = "Path to the controller's default identity client cert"
 	PkiCtrlServerCertVarName                         = "ZITI_PKI_CTRL_SERVER_CERT"
-	PkiCtrlServerCertVarDescription                  = "Path to Identity Server Cert for Ziti Controller"
+	PkiCtrlServerCertVarDescription                  = "Path to the controller's default identity server cert, including partial chain"
 	PkiCtrlKeyVarName                                = "ZITI_PKI_CTRL_KEY"
-	PkiCtrlKeyVarDescription                         = "Path to Identity Key for Ziti Controller"
+	PkiCtrlKeyVarDescription                         = "Path to the controller's default identity private key"
 	PkiCtrlCAVarName                                 = "ZITI_PKI_CTRL_CA"
-	PkiCtrlCAVarDescription                          = "Path to Identity CA for Ziti Controller"
+	PkiCtrlCAVarDescription                          = "Path to the controller's bundle of trusted root CAs"
 	CtrlBindAddressVarName                           = "ZITI_CTRL_BIND_ADDRESS"
-	CtrlBindAddressVarDescription                    = "The address on which the controller will listen on for control plane connections"
+	CtrlBindAddressVarDescription                    = "The address on which the controller will listen on for router control plane connections"
 	CtrlAdvertisedAddressVarName                     = "ZITI_CTRL_ADVERTISED_ADDRESS"
-	CtrlAdvertisedAddressVarDescription              = "The address routers will use to connect to the Ziti Controller"
+	CtrlAdvertisedAddressVarDescription              = "The address routers will use to connect to the controller"
 	CtrlAdvertisedPortVarName                        = "ZITI_CTRL_ADVERTISED_PORT"
-	CtrlAdvertisedPortVarDescription                 = "The port routers will use to connect to the Ziti Controller"
+	CtrlAdvertisedPortVarDescription                 = "The port routers will use to connect to the controller"
 	CtrlEdgeBindAddressVarName                       = "ZITI_CTRL_EDGE_BIND_ADDRESS"
-	CtrlEdgeBindAddressVarDescription                = "The address on which the edge controller will listen on for API connections"
+	CtrlEdgeBindAddressVarDescription                = "The address on which the controller will listen on for API connections"
```

Comment:
the word edge is important here. put it back pls. since you're touching this file, would you uncapitalize Ziti-related too.

Reply:
I could not find any occurrences of strings like "Ziti related" or "Ziti-related" in the option var descriptions in this file, constants.go, that I could uncapitalize. The closest thing I found were the Go variable names, but I doubt that's what you meant.